client/src/components/gameModeSettings/OutlineSelector.tsx

```diff
@@ -199,9 +199,9 @@ export function OutlineListSelector(props: {
 
     return <section className="graveyard-menu-colors selector-section">
         <h2>{translate("menu.lobby.roleList")}: {roleList.length}</h2>
-        <button disabled={props.disabled} onClick={simplify}>
+        {!props.disabled && <button onClick={simplify}>

```

Comment:
i would always slap a === true or === false on these and also capital b Button not button

Reply:
nitpick yknow